transifex/pull.py

```diff
@@ -223,7 +229,7 @@ def pull(repo):
                 logger.info('No changes detected for [%s]. Cleaning up.', repo.name)
 
         if pr:
-            if not repo.is_compiled:
+            if (not skip_compile_messages) and (not repo.is_compiled):
```

Comment:
Changed `is_compiled` to `compilemessages_failed`, since it is only used to flag when message compilation fails. That makes this if statement a lot clearer.